### PR TITLE
add encryption functionallty to ekm signer

### PR DESCRIPTION
**Pull request by guym-blox**

### CODE REVIEW

File: cli/operator/node.go [hunk 1]
```diff
@@ -97,7 +99,13 @@ var StartNodeCmd = &cobra.Command{
 		}
 		nodeStorage, operatorData := setupOperatorStorage(logger, db)
 
-		keyManager, err := ekm.NewETHKeyManagerSigner(logger, db, networkConfig, cfg.SSVOptions.ValidatorOptions.BuilderProposals)
+		operatorKey, _, err := nodeStorage.GetPrivateKey()
```
Comment (y0sher) [inline cli/operator/node.go:102]: check err

File: ekm/signer_key_manager_test.go [hunk 2]
```diff
@@ -47,6 +58,52 @@ func testKeyManager(t *testing.T) spectypes.KeyManager {
...
+	encryptionKey := fmt.Sprintf("%x", hash)
+	threshold.Init()
+	sk := bls.SecretKey{}
+	sk.SetByCSPRNG()
+	index := 0
+	logger := logging.TestLogger(t)
+	db, err := getBaseStorage(logger)
+	require.NoError(t, err)
+	signerStorage := NewSignerStorage(db, networkconfig.TestNetwork.Beacon, logger)
+	err = signerStorage.SetEncryptionKey(encryptionKey)
```
Comment (y0sher) [inline ekm/signer_key_manager_test.go:76]: check no err

File: cli/operator/node.go (hunk 1 above)
Comment (y0sher) [reply cli/operator/node.go:102]: this is still not solved

**Comment by y0sher** [comment]
Some tests fail, you probably need to fix up old tests to pass in a password.

File: cli/operator/node.go [hunk 3]
```diff
@@ -97,7 +99,12 @@ var StartNodeCmd = &cobra.Command{
 		}
 		nodeStorage, operatorData := setupOperatorStorage(logger, db)
 
-		keyManager, err := ekm.NewETHKeyManagerSigner(logger, db, networkConfig, cfg.SSVOptions.ValidatorOptions.BuilderProposals)
+		operatorKey, _, _ := nodeStorage.GetPrivateKey()
```
Comment (y0sher) [inline cli/operator/node.go:102]: I would say lets check for error but after `setupOperatorStorage` we should have it for sure.

**Review by y0sher — APPROVED** [review]

**Comment by moshe-blox** [comment]
@guym-blox do you think there is a point in encrypting/decrypting in `OpenWallet` and `SaveWallet` too?

File: cli/operator/node.go (hunk 3 above)
Comment (guym-blox) [reply cli/operator/node.go:102]: this is why I removed it,
in that step the private key must be set or the node won't run

**Review by moshe-blox — APPROVED** [review]

**Comment by moshe-blox** [comment]
gj 💪

**Pull request merged**